front-end/bundle-configuration/default/runtime-config.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+export APPLICATION_SECRET=PPjOW0n2aV?s@6RdiNV@7/5xJhiiKTzk[VdHjkU9YHit8sLHoJ1rp0DCCn6b=lXt
```

Comment:
I don't think so. We are forcing users to think about alternate config they must perform a deliberate action to load it - and if they don't load the config then changeme will be there. Make sense?